core/controllers/reader.py

```diff
@@ -987,6 +987,11 @@ def get(self):
             raise self.InvalidInputException(
                 'Question count has to be greater than 0')
 
+        if int(question_count) > int(constants.MAX_QUESTIONS_IN_QUESTION_PLAYER):
+            raise self.InvalidInputException(
+                'Question count has to be less than %d',
+			%(constants.MAX_QUESTIONS_IN_QUESTION_PLAYER))
```

Comment:
Remove `,` between `'` and `%` also if not required, remove the unwanted parenthesis.